### PR TITLE
akc020's hometown file 

### LINKED ISSUE #10: akc020 checklist
- [x] Create an issue
- [x] Create branch
- [x] Create a PR
- [x] Collaborate
- [x] Merge

### PR DESCRIPTION
**Pull request by akc020**
adding Orlando, FL with my recommendations.

Closes #10

### CODE REVIEW

File: akc020-orlando.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Orlando, FL
+
+## Places to Eat
+
+- Frank N Steins :hamburger:
+- Jungle Jim's :hamburger:
```
Comment (gmstrbytes) [inline akc020-orlando.md]: You have provided hamburger twice. Could we have some sushi instead please?

File: akc020-orlando.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Disney Springs :mouse:
```
Comment (gmstrbytes) [inline akc020-orlando.md]: How does the mouse compare with the grilled squirrel?

Comment (gmstrbytes) [inline akc020-orlando.md]: How does the mouse compare with the grilled squirrel?

File: akc020-orlando.md (hunk 1 above)
Comment (gmstrbytes) [inline akc020-orlando.md]: You have provided hamburger twice. Could we have some sushi instead please?

**Review by gmstrbytes — DISMISSED** [review]
Please provide more diversity

**Review by beardofedu — APPROVED** [review]

**Pull request merged**